Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM bash:4.4
+
+RUN addgroup -g 1000 testing && \
+    adduser -u 1000 -G testing -s /usr/local/bin/bash -D testing
+
+RUN apk --no-cache add \
+    curl \
+    git \
+    jq \
+    libc6-compat \
+    openssh-client \
+    python \
+    py-crcmod \
+    py-pip
+
+RUN STABLE=$(curl -Ls https://storage.googleapis.com/kubernetes-release/release/stable.txt) && \
```

Comment:
Lets hard code the version so we know whats in the image based on the commit/tag